pkg/devconsole/tekton-results/results.go

```diff
@@ -94,21 +94,27 @@ func makeHTTPRequest(url, userToken string) (common.DevConsoleCommonResponse, er
 
 	// Load the CA certificate
 	caCert, err := os.ReadFile(tlsCertPath)
-	if err != nil {
+	if err != nil && k8sMode != "off-cluster" {
```

Comment:
It would be better to just skip trying to read the file if off-cluster instead of reading it, failing, and ignoring errors.

Reply:
Updated.